### ISSUE TITLE
[Question] How this library should be named ?

### ISSUE THREAD

**Issue by FranckRJ** [boost-ext/di#476]
Hello, I wanted to add this library into conan-center ( https://github.com/conan-io/conan-center-index/pull/1855 ) but they're not sure how it should be named. Boost authors don't want it to be named "boost-di" because it's not a boost library. Have you any "official" suggestion on how we should call your library without having any conflict with boost ? Thanks !

**Comment by kris-jusiak** [comment]
Is just `di`, okay/available?

On vcpkg it is https://github.com/microsoft/vcpkg/tree/master/ports/boost-di

**Comment by Croydon** [comment]
On vcpkg it seems to be `boost-di` not just `di`

**Comment by kris-jusiak** [comment]
Yeah, `boost-di` would be great, wasn't sure if that's an option?
1. boost-di
2. di
3. ?

**Comment by danimtb** [comment]
there seems to be some controversy with the name "boost" regarding the boost organization. Also, just the name "di" is not very descriptive and not so easy to relate to this project 🤔

**Comment by kanstantsin-chernik** [comment]
I think it is ok. There is any other boost di in this world. Also, boost di is fully compatible with boost and implemented with the same license and standards. Of course, it could be a part of boost, but as it is open-source they are reallllly slow...
